frontend/src/js/common-ui/DeviceIdentity.tsx

```diff
@@ -64,7 +64,7 @@ export const getDeviceIdentityText = ({ device = {}, idAttribute }) => {
   const { attribute, scope } = idAttribute;
   // special handling for tags purely to handle the untagged devices case
   if (attribute === 'name' && scope === 'tags') {
-    return tags[idAttribute] ?? `${id.substring(0, 6)}...`;
+    return tags[attribute] ?? `${id.substring(0, 6)}...`;
```

Comment:
Maybe it's better to fallback to `-` so it's less misleading?

Reply:
that's according to the design (to have at least a reference to the device id in a device list if there's no name configured), unfortunately the old design with the definition for this is no longer accessible, but https://northerntech.atlassian.net/browse/MEN-4157?focusedCommentId=59511 still has _some_ reference to this 👌